### DISCUSSION TITLE
<AI音樂討論>  人與AI樂團玩奏出新花樣

### DISCUSSION THREAD

**Discussion by EricAgumonJou** [1101-GS4719-C/Python-Community-GS4719-C#308] [General]
我想分享 AI應用於音樂創作領域的一篇文章。

清大該AI樂團，運用AI譜出新曲及自動跟譜技術，讓表演者演奏到哪，軟體便自動追蹤音樂內容，創造真人與AI軟體同台演出的意境。

想詢問同學們對於AI音樂創作抱持著什麼樣的想法？
未來你們覺得AI音樂創作會越來越普及嗎？

https://www.parenting.com.tw/article/5080956

**Comment by EricAgumonJou** [comment]
我認為AI作曲固然能讓各個音樂創作者突破瓶頸。
我自己在聽其他的AI音樂時，聽感還是有種說不上來的毛毛感，不知是不是因為自己沒聽過好聽的AI音樂的關係。😥

**Reply by EricAgumonJou** [reply]
也有可能是因為，太過完美的AI音樂 讓人感覺不出音樂的美感與溫度。

**Comment by 20010906** [comment]
AI會不會經過不斷的訓練及學習後，懂得如何挑選出具美感及溫度的音樂？
